### ISSUE TITLE
G0 - Bootstrap Alerts fail contrast test

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#185]
Seems like the bootstrap alerts are still failing the colour contrast test. 

https://aoda.web.uoguelph.ca/aodademo/stylesheet

**Comment by mmafe** [comment]
So far, I'm currently reading:

Green: text #057A51 on background #DFF0D8 = 4.5 contrast
Blue: text #336F98 on background #D9EDF7 = 4.5 contrast
Yellow: text #A66208 on background #FCF8E3 = 4.5 contrast
Red: text #B83635 on background #F2DEDE = 4.5 contrast

These all pass the success criteria. Are these the same values that you're testing? Let me know if you're picking up different values and we can check them.

**Comment by jpinsent** [comment]
Oh, you're right. I just double checked and it looks like they do pass. I think I was looking at it in the WYSIWYG editor where it shows up with the default gradient styling and got confused.
